server/container/guice/cassandra-guice/src/test/java/org/apache/james/DockerCassandraRule.java

```diff
@@ -43,15 +46,17 @@ public void await() {
 
     @Override
     public Module getModule() {
-        String keyspace = new RandomStringGenerator.Builder().withinRange('a', 'z').build().generate(12);
-        return (binder) -> binder.bind(ClusterConfiguration.class)
+        return Modules.combine((binder) -> binder.bind(ClusterConfiguration.class)
             .toInstance(ClusterConfiguration.builder()
                 .host(cassandraContainer.getHost())
-                .keyspace(keyspace)
+                .keyspace("testing")
```

Comment:
is it a duplicated String value ?

Reply:
God damn no! I'm not that kind of criminal! It's two indepandant definitions of keyspaces that happens to have the same name, that's all...